src/assignment/views/AssignmentDetail.tsx

```diff
@@ -281,9 +291,123 @@ const AssignmentDetail: FC<DefaultSecureRouteProps<{ id: string }>> = ({
 		}
 	};
 
+	const onEditAssignment = () => {
+		history.push(
+			generatePath(APP_PATH.ASSIGNMENT_EDIT_TAB.route, {
+				id,
+				tabId: ASSIGNMENT_CREATE_UPDATE_TABS.CONTENT,
+			})
+		);
+	};
+
+	const onDuplicateAssignment = async (): Promise<void> => {
+		try {
+			if (!assignment) {
+				ToastService.danger(
+					tHtml(
+						'De opdracht kan niet gekopieerd worden omdat deze nog niet is opgehaald van de database'
+					)
+				);
+				return;
+			}
+
+			if (!user.profile) {
+				ToastService.danger(
+					tHtml(
+						'Er was een probleem met het controleren van de ingelogde gebruiker, log opnieuw in en probeer opnieuw'
+					)
+				);
+				return;
+			}
+
+			const duplicate = await duplicateAssignment(assignment, user.profile.id);
+			if (duplicate) {
+				history.push(
+					generatePath(APP_PATH.ASSIGNMENT_DETAIL.route, {
+						id: duplicate.id,
+					})
+				);
+				setAssignment(duplicate as Avo.Assignment.Assignment);
+			}
+
+			ToastService.success(tHtml('De opdracht is gekopieerd u kijkt nu naar de kopie'));
```

Comment:
gebruik altijd "je". blijkbaar mag "u" niet meer gebruikt worden volgens meemoo
```suggestion
			ToastService.success(tHtml('De opdracht is gekopieerd. Je kijkt nu naar de kopie'));
```